mobify/progressive-web-scaffold
# Image Optimization Hotfix

Move image optimization that was done in `develop` branch and hotfix insert it into master (we don't want to fully merge `develop` into master yet)

## Changes
- Optimize home page carousel images

## How to test-drive this PR
- Checkout and pull down updates from `image-hotfix` branch
- Delete modules with `rm -rf node_modules` from the project root
- Run `npm i` to install this older set of dependencies
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Verify that the homepage image assets have been reduced to 30-50kb size range instead of 500kb

## Todo
- [ ] After merging, cut a new release!
